Modules/Module_Import/Import/xASL_imp_NII2BIDS_Run.m

```diff
@@ -70,29 +69,29 @@
 		end
 	catch loggingEntry
 		[x] = xASL_qc_AddLoggingInfo(x, loggingEntry);
-		xASL_imp_NII2BIDS_RunIssueWarning(x,loggingEntry, 'anatomical', subjectSessionLabel, iRun);
+		xASL_imp_NII2BIDS_RunIssueWarning(loggingEntry, 'anatomical', subjectSessionLabel, iRun);
 	end
 		
 	%% Perfusion files
 	try
 		xASL_imp_NII2BIDS_RunPerf(x.modules.import.imPar, bidsPar, studyPar, subjectSessionLabel, inSessionPath, outSessionPath, listRuns, iRun);
 	catch loggingEntry
 		[x] = xASL_qc_AddLoggingInfo(x, loggingEntry);
-		xASL_imp_NII2BIDS_RunIssueWarning(x,loggingEntry, 'perfusion', subjectSessionLabel, iRun);
+		xASL_imp_NII2BIDS_RunIssueWarning(loggingEntry, 'perfusion', subjectSessionLabel, iRun);
 	end
 
 end
 
+%% -------------------------------------------------------------
 %% Issue failed run as a warning
-function xASL_imp_NII2BIDS_RunIssueWarning(x,loggingEntry, Scantype, subjectSessionLabel, iRun)
+%% -------------------------------------------------------------
+function xASL_imp_NII2BIDS_RunIssueWarning(loggingEntry, Scantype, subjectSessionLabel, iRun)
 
     xASL_adm_BreakString('');
-    fprintf(2,'NII2BIDS failed for %s image of %s_run-%s\n',Scantype,subjectSessionLabel,xASL_num2str(iRun));
+    fprintf(2,'NII2BIDS failed for %s image of %s_run-%s\n', Scantype, subjectSessionLabel, xASL_num2str(iRun));
```

Comment:
If you're revamping this, then also add a space after the comma before 'NII2BIDS ;) same next line

Reply:
Oops. I've overlooked that. Done.